src/yaffshiv

```diff
@@ -5,6 +5,14 @@ import sys
 import struct
 import string
 
+def is_safe_path(basedir, path, follow_symlinks=True):
```

Comment:
Why the `follow_symlinks` parameter when it is never called with `False`?

Reply:
Agreed. Fixed.